src/Components/ServiceScene/LogsTableScene.tsx

```diff
@@ -52,7 +52,7 @@ export class LogsTableScene extends SceneObjectBase<LogsTableSceneState> {
 
     // Define callback function to update url columns in react
     const setUrlColumns = (urlColumns: string[]) => {
-      if (!areArraysEqual(urlColumns, parentModel.state.urlColumns)) {
+      if (!areArraysEqualAndHaveSameOrder(urlColumns, parentModel.state.urlColumns)) {
```

Comment:
```suggestion
      if (!areArraysStrictlyEqual(urlColumns, parentModel.state.urlColumns)) {
```

Or adding options to `areArraysEqual`.